python/kserve/poetry.lock

```diff
@@ -2,98 +2,98 @@
 
 [[package]]
 name = "aiohttp"
-version = "3.8.4"
```

Comment:
I think these changes should be a separate PR?

Reply:
I can either prepare another PR, or rearrange this PR into two commits.
Whatever works best for you.

Reply:
OK, I did the split.

Now, this PR has only the python updates.
I opened another PR containing the updates of the go-code: #3245

Reply:
...and after rebasing, looks like all security issues were fixed in the ray update PR. So, this PR is no longer needed. Only the other PR for the go-code is still relevant.